kubectl-minio/cmd/tenant-create.go

```diff
@@ -143,34 +158,30 @@ func createTenant(oclient *operatorv1.Clientset, kclient *kubernetes.Clientset,
 	if err != nil {
 		return err
 	}
+	minSvc := services.NewClusterIPForMinIO(to)
+	conSvc := services.NewClusterIPForConsole(to)
 	if color.IsTerminal() {
-		consolePort := miniov1.ConsolePort
-		minioPort := miniov1.MinIOPort
-		if to.HasCertConfig() || to.AutoCert() {
-			consolePort = miniov1.ConsoleTLSPort
-		}
-		printBanner(to.ObjectMeta.Name, string(console.Data["CONSOLE_ACCESS_KEY"]), string(console.Data["CONSOLE_SECRET_KEY"]),
-			to.ConsoleCIServiceName(), to.MinIOHLServiceName(), consolePort, minioPort, (to.HasCertConfig() || to.AutoCert()))
+		printBanner(to.ObjectMeta.Name, to.ObjectMeta.Namespace, string(console.Data["CONSOLE_ACCESS_KEY"]), string(console.Data["CONSOLE_SECRET_KEY"]),
+			minSvc, conSvc)
 	}
 	return nil
 }
 
-func printBanner(tenantName, user, pwd, consoleSVCName, minioSVCName string, consolePort, minioPort int, tls bool) {
-	minioLocalPort := 9000
-	consoleLocalPort := 9090
-	scheme := "http"
-	if tls {
-		scheme = "https"
-	}
-	fmt.Printf(color.Bold(fmt.Sprintf("\nMinIO Tenant '%s' created\n\n", tenantName)))
-	fmt.Printf(color.Blue("Username: ") + color.Bold(fmt.Sprintf("%s \n", user)))
-	fmt.Printf(color.Blue("Password: ") + color.Bold(fmt.Sprintf("%s \n\n", pwd)))
-
-	fmt.Printf(color.Blue("Web interface access: \n"))
-	fmt.Printf(fmt.Sprintf("\t$ kubectl port-forward svc/%s %d:%d\n", consoleSVCName, consoleLocalPort, consolePort))
-	fmt.Printf(fmt.Sprintf("\tPoint browser to %s://localhost:%d\n\n", scheme, consoleLocalPort))
-
-	fmt.Printf(color.Blue("Object storage access: \n"))
-	fmt.Printf(fmt.Sprintf("\t$ kubectl port-forward svc/%s %d:%d\n", minioSVCName, minioLocalPort, minioPort))
-	fmt.Printf((fmt.Sprintf("\t$ mc alias set %s %s://localhost:%d %s %s\n\n", tenantName, scheme, minioLocalPort, user, pwd)))
+func printBanner(tenantName, ns, user, pwd string, s, c *corev1.Service) {
+	fmt.Printf(color.Bold(fmt.Sprintf("\nTenant '%s' created in '%s' Namespace\n\n", tenantName, ns)))
```

Comment:
Ah, sorry one more thought @nitisht 

We should warn that these credentials are only shown once. Something like

"Copy the following administrator credentials to a secure location, such as password-protected key storage. MinIO does *not* show these credentials again"

Reply:
even if technically they can extract from secrets, better to not force users down that path :)

Reply:
done